language/move-prover/tests/testsuite.rs

```diff
@@ -254,41 +254,39 @@ fn get_flags_and_baseline(
 fn collect_enabled_tests(reqs: &mut Vec<Requirements>, group: &str, feature: &Feature, path: &str) {
     let mut p = PathBuf::new();
     p.push(path);
-    for e in WalkDir::new(p.clone()).min_depth(1).into_iter() {
-        if let Ok(entry) = e {
-            if !entry.file_name().to_string_lossy().ends_with(".move") {
-                continue;
-            }
-            let path = entry.path();
-            let mut included = match feature.inclusion_mode {
-                InclusionMode::Implicit => !extract_test_directives(path, "// exclude_for: ")
-                    .unwrap_or_default()
-                    .iter()
-                    .any(|s| s.as_str() == feature.name),
-                InclusionMode::Explicit => extract_test_directives(path, "// also_include_for: ")
+    for entry in WalkDir::new(p.clone()).min_depth(1).into_iter().flatten() {
```

Comment:
Same here if this was a lint, though I don't expect push back. This seems much better.